spec/library_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'yaml'
+require './lib/library.rb'
+
+#collection = YAML.load_file('./lib/booksdb.yml')
+
+describe Library do
+  let(:patron) {double('Patron')}
+
+  it 'should have the list of books' do
+    expect(subject.bookrack).not_to be nil
+  end
+
+  it 'should check if the book is available' do
+     expect(subject.bookrack[0][:available]).to be true
+  end
```

Comment:
This test should be seeing if a method works correctly. Here you are only testing that the `bookrack` is receiving the contents of your yaml file (which you are doing implicitly in the method above).